.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,4 +11,6 @@ Before you submit this PR:
 - Make sure your tests pass
 - Include screenshots if necessary but do so on the JIRA page not here.
```

Comment:
i was thinking it would be good to have a sort of checklist when you're creating a PR. So you know you've covered everything.